config/utils.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func getConfigYaml(path string) ([]byte, error) {
+	if strings.HasPrefix(path, "http") {
+		res, err := http.Get(path)
+		if err != nil {
+			return []byte{}, err
+		}
+
+		if res.StatusCode != 200 {
```

Comment:
You can use http status from std lib here like:
```suggestion
		if res.StatusCode != http. StatusOK {
```